### ISSUE TITLE
[FR] Hold Action - Lock 

### ISSUE THREAD

**Issue by WolfsWelpe** [dwainscheeren/dwains-lovelace-dashboard#226]
Hey Dwain,

Just a little thing. Could you maybe implement something to call the service lock.open ?
Something like when you hold it down and release it or double tap or something like that.
For now you are just calling toggle I think ? Which is only unlocking and locking the door without opening it. :/ 
Would be very nice <3 

Have a nice day and maybe look in your PayPal, gifted you a little bit few weeks ago :) 
Thank you for 2.0 ^^ And thank you so much for your Support :)

**Comment by groft389** [comment]
Addition;
As for now I changed in rooms.yaml (dwains_dashboard/lovelace/views/main/rooms/room.yaml) this in line 233:
```
                hold_action:
                  action: call-service
                  service: lock.open
                  service_data:
                    entity_id: {{ room["lock"] }}
```

**Comment by groft389** [comment]
For now this works fine.
Maybe my feature request is too specific idk.
But it would be nice at least for locks to enter a ping or something for unlocking or opening ? 
Maybe this is too much for the moment ^^ You just released 2.0 and Im so thankful you still working 
on this project. Earlier I was so scared you would abandon this project :O

Go on like that :)
